src/styles/_catalog.scss

```diff
@@ -14,6 +16,14 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    cursor: pointer;
+
```

Comment:
styles properties order https://codeguide.academy/html-css.html#css-order

Reply:
I did not understand. Cursor changing seems to be amimation or others. Previous properties were written by mentors, so do they have an issue?)

Reply:
margin I mean must be higher